docs/content/latest/integrations/kairosdb.md

```diff
@@ -73,32 +70,139 @@ Use HELP for help.
 ycqlsh>
 ```
 
-- Run the following YCQL commands to verify it is working:
+- Run the following YCQL commands, and connect to the `kairosdb` keyspace to verify it is working:
 
-```sql
+```cql
 ycqlsh> describe keyspaces;
 ```
 
 ```output
 kairosdb  system_schema  system_auth  system
 ```
 
-```sql
+```cql
 ycqlsh> use kairosdb;
+```
+
+```cql
 ycqlsh:kairosdb> describe tables;
 ```
 
 ```output
-row_keys       data_points    string_index
-row_key_index  service_index  row_key_time_index
+tag_indexed_row_keys  row_key_index  service_index  row_key_time_index
+row_keys              data_points    string_index   spec
 ```
 
-## 5. Test KairosDB
+## Test KairosDB
+
+- Start Postman and create a new workspace.
 
 ### Push data
 
-Push metric data into KairosDB as per the instructions [here](https://kairosdb.github.io/docs/PushingData.html).
+- Select the POST API request and add the following URL in the `Enter request URL` box:
+
+```text
+http://localhost:8080/api/v1/datapoints
+```
+
+- In the body of the request, add the following JSON, and send it.
+
+```json
+[
+  {
+      "name": "archive_file_tracked",
+      "datapoints": [[1359788400000, 123], [1359788300000, 13.2], [1359788410000, 23.1]],
+      "tags": {
+          "host": "server1",
+          "data_center": "DC1"
+      },
+      "ttl": 300.0
+  },
+  {
+      "name": "archive_file_search",
+      "timestamp": 1359786400000,
+      "value": 321,
+      "tags": {
+          "host": "server2"
+      }
+  }
+]
+```
+
+Your request should look like:
+
+![kairosdb POST request](/images/develop/ecosystem-integrations/kairosdb/kairosdb1.png)
```

Comment:
Possible to resize the windows of these screenshots so they doesn't show all that whitespace? They would then be easier to read, being less wide.

Reply:
Replaced it with the ones that showed on my Postman UI.